app/Helper.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+class Helper
+{
+
+    public static function transformDate($date, $format = 'd-m-Y')
+    {
+        return date($format, strtotime($date));
```

Comment:
Use Carbon library if you work with dates

Reply:
Done